src/index.ts

```diff
@@ -45,6 +45,36 @@ export default function rdme(processArgv: NodeJS.Process['argv']) {
     { name: 'command', type: String, defaultOption: true },
   ];
 
+  let processArgv = rawProcessArgv;
+
+  /**
+   * We have a weird edge case with our Docker image version of `rdme` where GitHub Actions
+   * will pass all of the `rdme` arguments as a single string with escaped quotes,
+   * as opposed to the usual array of strings that we typically expect with `process.argv`.
+   *
+   * For example, say the user sends us `rdme openapi "petstore.json"`.
+   * Instead of `process.argv` being this (i.e., when running the command via CLI):
+   * ['openapi', 'petstore.json']
+   *
+   * The GitHub Actions runner will send this to the `rdme` Docker image:
+   * ['openapi "petstore.json"']
+   *
+   * To distinguish these, we have a hidden `docker-gha` argument which we check for to indicate
+   * when arguments are coming from the GitHub Actions runner.
+   * This logic checks for that `docker-gha` argument and parses the second string
+   * into the arguments array that `command-line-args` is expecting.
+   */
+  if (rawProcessArgv.length === 2 && rawProcessArgv[0] === 'docker-gha') {
```

Comment:
We now check for that `docker-gha` argument before parsing out the second string and converting it to something that `command-line-args` can process.

Reply:
Hopefully this is something we can remove down the line! But nice fix for now.